domains/george.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "George Aykara's personal developer website",
+    "repo": "https://github.com/aykarageorge",
+    "owner": {
+        "username": "aykarageorge",
+        "email": "aykarageorge@gmail.com"
+    },
+    "record": {
+        "URL": "https://george.aykara4.com"
```

Comment:
URL redirection is currently facing a bug (#221). This could take a week or so to be fixed. I'd suggest using a cname record instead. Let me know if you want to keep it this way, then I'll merge it.

Reply:
Updated to CNAME. Thanks.